ui/src/views/admin/settings/SettingsView.vue

```diff
@@ -1,82 +1,145 @@
 <template>
   <v-container
     id="admin-setting-view"
-    fluid
     tag="section"
   >
     <v-row justify="center">
-      <v-col
-        cols="8"
-        md="8"
-      >
-        <material-card
-          color="primary"
+      <v-col :cols="isMobile? 12 : 8">
+        <v-card
+          class="admin-settings-view__card"
+          :elevation="isMobile? 0 : 2"
         >
-          <v-img
-            class="rounded-circle mt-n12 d-inline-block"
-            :src="profilePicture"
-            width="128"
-          />
-          <v-form>
-            <v-container class="py-0">
-              <v-row>
-                <v-col
-                  cols="12"
-                  md="12"
-                >
-                  <v-text-field
-                    v-model="formData.email"
-                    disabled
-                    label="E-mail address"
-                  />
-                </v-col>
-
-                <v-col
-                  cols="12"
-                  md="12"
-                >
-                  <v-text-field
-                    v-model="formData.password"
-                    color="purple"
-                    label="Password"
-                  />
-                </v-col>
-
-
-                <v-col
-                  cols="12"
-                >
-                  <v-btn
-                    color="primary"
-                    min-width="150"
+          <v-row justify="center">
+            <v-col cols="4">
+              <v-img
+                class="rounded-circle d-inline-block mx-2 mb-5"
+                :src="require('@assets/svg/profile_picture.svg')"
+                width="128"
```

Comment:
The pic size in Figma is bigger, around 200-214. You can check the pic size by clicking on the image (I assumed you have an acc already) and it should be displayed 
![image](https://user-images.githubusercontent.com/12537724/172919015-d105b14e-a010-41f9-bb54-0f8b1ac0c85e.png)